<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeImpSelectPage</class>
 <widget class="QWidget" name="CodeImpSelectPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>539</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Code Generation Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="ui_topLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="ui_directoryLayout">
       <item>
        <widget class="QLabel" name="ui_URLRequesterLabel">
         <property name="toolTip">
          <string>The base directory that contains the project</string>
         </property>
         <property name="text">
          <string>&amp;Directory of the project:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="buddy">
          <cstring>ui_kURL</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KUrlRequester" name="ui_kURL">
         <property name="toolTip">
          <string>The base directory that contains the project</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="ui_subdirCheckBox">
       <property name="text">
        <string>Include Subdirectories</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="ui_languageLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="ui_languageLabel">
         <property name="toolTip">
          <string>The language of the project</string>
         </property>
         <property name="text">
          <string>Programming Language:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KComboBox" name="ui_languageBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="ui_treeView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ui_commandButtonLayout">
     <item>
      <widget class="QPushButton" name="ui_deselectAllButton">
       <property name="text">
        <string>Deselect all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_selectAllButton">
       <property name="text">
        <string>Select all</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="ui_labelLayout">
       <property name="formAlignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="ui_filesLabel">
         <property name="text">
          <string>Number of files:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="ui_filesNumLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">knuminput.h</include>
  <include location="local">knuminput.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
